drivers/scheduler/dcos/specs/mysql/px-mysql.json

```diff
@@ -0,0 +1,50 @@
+{
+    "id": "/pxmysql",
+    "instances": 1,
+    "cpus": 0.5,
+    "mem": 256,
+    "backoffSeconds": 1,
+    "backoffFactor": 1.15,
+    "maxLaunchDelaySeconds": 3600,
+    "container": {
+        "type": "DOCKER",
+        "docker": {
+            "image": "mysql:5.7.12",
+            "network": "BRIDGE",
+            "portMappings": [
+                {
+                    "containerPort": 3306,
+                    "hostPort": 32000,
+                    "servicePort": 10002,
+                    "protocol": "tcp",
+                    "name": "default"
+                }
+            ],
+            "privileged": false,
+            "parameters": [
+                {
+                    "key": "volume-driver",
+                    "value": "pxd"
+                },
+                {
+                    "key": "volume",
+                    "value": "size=10,repl=3,name=px_mysql_vol:/var/lib/mysql"
```

Comment:
reduce size to 2. 10G will cause issues on dev systems which might have less storage.

Reply:
done